src/components/HeaderNavMenu.jsx

```diff
@@ -21,7 +21,8 @@ function HeaderNavMenu({ links }) {
       indicatorColor="primary"
       scrollButtons="auto"
       value={currentTab}
-      variant="scrollable">
+      variant="scrollable"
+      sx={{ display: 'inline-flex' }}>
```

Comment:
What does this do?

Reply:
It prevents the following. 
<img width="382" alt="Screen Shot 2022-01-07 at 8 38 14 PM" src="https://user-images.githubusercontent.com/40483569/148631605-a1226c70-8206-46e1-8750-d06f15898a94.png">

meanwhile `display: inline` causes the following:
<img width="472" alt="Screen Shot 2022-01-07 at 8 39 14 PM" src="https://user-images.githubusercontent.com/40483569/148631629-4e4e84fe-4db1-4bc8-a36b-4e5b77cc98e1.png">